PIM/UserInterface/02a_List.md

```diff
@@ -0,0 +1,1419 @@
+
+> [Info] If more than one defining attribute is assigned to the variant, you have to select a value for each defining attribute to add a variant.
+
+- Defining attribute(s)   
+  Enter or select the appropriate value for the defining attribute(s).
+
+- [CANCEL]   
+  Click this button to cancel adding a variant and close the *Add single variant* window.
+
+- [SAVE]   
+  Click this button to save the variant and close the *Add single variant* window.
+
+
+### Add multiple variants
+*PIM > Products > Tab LIST > Button Add > Button CREATE > Tab Variants > Select variant set > Button Add > Button Add multiple variants*
+
+#### Select Values
+
+![Add multiple variants](../../Assets/Screenshots/PIM/Products/List/Variants/AddMultipleVariants01.png "[Add multiple variants]")
+
+For each defining attribute a single box to define the attribute value is displayed.
+
+> [Info] If more than one defining attribute is assigned to the variant, you have to select a value for each defining attribute to add a variant.
+
+- [ADD ALL VALUES]   
+  Click this button to add all available values in the box for the corresponding defining attribute. This button is only displayed for attributes with the *TreeNode* data type.
+
+- [ADD VALUE]   
+  Click this button to add a further row to the corresponding attribute where you can enter or select the desired value.
+
+- [ABORT]   
+  Click this button to cancel adding variants and close the *Select Values* wizard window.
+
+- [NEXT]   
+  Click this button to proceed to the next step of adding variants. The *Summary* wizard window is displayed.
+
+
+#### Summary
+
+![Add multiple variants](../../Assets/Screenshots/PIM/Products/List/Variants/AddMultipleVariants02.png "[Add multiple variants]")
+
+The box displays all variants that will be created from the selected attribute values. For each possible combination of the selected attributes, a new variant is created. Variants that already exist will not be created even when they are listed in the box.
+
+**Variants that will be created (ignoring duplicated)**
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all variants in the list are selected.
+
+- ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete)   
+  Click this button to delete the selected variant from the list. This button is only displayed, when the checkbox of a variant is selected.
+
+The list displays all selected variants. Depending on the settings, the displayed columns may vary.
+
+- *SKU*      
+  Stock Keeping Unit. Identification number for the variant. Click this field to edit the SKU of the selected variant.
+
+- Defining attribute     
```

Comment:
Wie sollen wir ein UI beschreiben, die keinen (festen) Namen hat, wie hier? In Klammern oder einfach runde Schrift? In Buchhaltung oft/immer in Klammern.